src/Keboola/GoogleSheetsWriter/Sheet.php

```diff
@@ -136,7 +121,7 @@ private function uploadValues(array $sheet, Table $inputTable) : array
                     if ($offset === 1) {
                         $sheetValues = $this->client->getSpreadsheetValues(
                             $sheet['fileId'],
-                            urlencode($sheet['sheetTitle'])
+                            $this->getRange($sheet['sheetTitle'], $inputTable->getColumnCount(), $offset, 100)
```

Comment:
Nechápu, co tohle dělá? Proč to tahá jen 100 řádků?

Reply:
Resp. hlavně proč to netahá jen jeden (tj. ten header)

Reply:
Ideálně bych to zabalil do popisné metody třeba `hasSheetHeader()`

Reply:
Jo to mas asi pravdu, ze by to mohlo tahat len header. 
Chcem to cele trochu prerobit, vybrat tu podmienku z toho cyklu niekam vyssie idealne a zabalit do metody je tiez dobry napad.